src/elements/content-sidebar/ContentSidebar.scss

```diff
@@ -27,14 +27,26 @@ $sidebarIncreasedWidth: $sidebarTabsWidth + $sidebarContentIncreasedWidth;
         white-space: nowrap;
     }
 
+    .be-default-error {
+        min-width: 324px;
```

Comment:
Can we create a variable for this value describing where it comes from, similiar to how it is done in https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/SidebarContent.scss?
```
$sidebarErrorWidth = $sidebarWidth - 16; // [Comment with explanation ]
```

And the same applies to the wider variant (line 42).